\documentclass[margin,line]{res}

\usepackage{hyperref}

\oddsidemargin -.5in
\evensidemargin -.5in
\textwidth=6.0in
\itemsep=0in
\parsep=0in
% if using pdflatex:
%\setlength{\pdfpagewidth}{\paperwidth}
%\setlength{\pdfpageheight}{\paperheight} 

\newenvironment{list1}{
  \begin{list}{\ding{113}}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.17in}}}{\end{list}}
\newenvironment{list2}{
  \begin{list}{$\bullet$}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.2in}}}{\end{list}}


\begin{document}

\name{Andrew Quitadamo \vspace*{.1in}}

\begin{resume}
\section{\sc Contact Information}
\vspace{.05in}
andrew.quitadamo@gmail.com \\

\section{\sc Education}
{\bf University of North Carolina at Charlotte}, Charlotte, North Carolina\\

\vspace*{-.1in}
\begin{list1}
\item[] Ph.D., Bioinformatics and Genomics, December 2018

\end{list1}

{\bf University of New Hampshire}, Durham, New Hampshire\\ 
\vspace*{-.1in}
\begin{list1}
\item[] B.S., Biochemistry, Molecular and Cellular Biology,  May 2013
\end{list1}

\section{\sc Experience}
{\bf Boston Children's Hospital}\\
{\em Bioinformatics Scientist} \hfill {\bf May 2019 - Present}\\
Helped maintain a codebase to coordinate a sequencing project for rare disease cohorts and curated variants. Worked with over a dozen diseases and investigators to find variants of interest in patients. 


\section{\sc Skills}
Experienced: Python, R, Git/GitHub, Unix/Linux Shell, \LaTeX\ \\
Familiar: Java, Matlab, Javascript, AWS


\section{\sc Research Experience}
{\bf University of North Carolina at Charlotte}\\
%\vspace{-.3cm}
{\em Shi Lab} \hfill {\bf January 2014 - December 2018}\\
%
Developing network models of eQTLs, and applied eQTL analysis techniques to cancer data.
Analyzed the impact of structural variants on gene expression.
Developed an eQTL analysis pipeline software package. 

%\vspace{-.1cm}
{\em Janies Lab} \hfill {\bf August 2013 - December 2013}\\
Created a phylogenetic tree of Phlebotominae sandflies using multiple genetic markers.    
\hspace*{.05in} 

{\bf University of New Hampshire}\\
%\vspace{-.3cm}
%\vspace{-.1cm}
{\em Thomas Lab} \hfill {\bf June 2012 - June 2013}\\
Undergraduate research assistant, studied DNA methylation in {\em C. elegans}.\\ 


\section{\sc Teaching Experience}

{\em Teaching Assistant} \hfill {\bf August 2017-December 2017}\\
BINF 6112/8112 - Bioinformatics Programming I\\ 
Co-taught Introduction to Bioinformatics Programming. Prepared and gave lectures on the command line and Python basics during the first month of the semester. 

{\em Teaching Assistant} \hfill {\bf January 2016-May 2016}\\
BINF 6112/8112 - Bioinformatics Programming II\\
Taught lab section of Python programming and guest lectured. Taught classes on text editors, Git and GitHub, unit testing and other subjects.

{\em Teaching Assistant} \hfill {\bf January 2015-May 2015}\\
BINF 6112/8112 - Bioinformatics Programming II\\
Taught lab section of Python programming. 

\section{\sc Honors and Awards} 

Essam El-Kwae Student-Faculty Research Award 2016

GAANN Fellowship 2016

University Scholar 2012, 2013

\section{\sc Selected Publications}

Xie R, Wen J, {\bf Quitadamo A}, Cheng J, Shi X. ``A deep auto-encoder model for gene expression prediction``. BMC Genomics 2017. 18(Suppl 9): 845. doi:10.1186/s12864-017-4226-0

Wen J, {\bf Quitadamo A}, Hall B, Shi X. ``Epistasis analysis of microRNAs on pathological stages in colon cancer based on an Empirical Bayesian Elastic Net method``. BMC Genomics 2017. 18(Suppl 7): 756. doi:10.1186/s12864-017-4130-7

Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, Abyzov A, Huddleston J, Zhang Y, Ye K, Jun G, Fritz M et al. The 1000 Genomes Project Consortium, Mills RE, Gerstein MB, Bashir A, Stegle O, Devine SE, Lee C, Eichler EE, Korbel JO. ``An integrated map of structural variation in 2,504 human genomes''. Nature 2015. 562: 75-81. doi:10.1038/nature15394

The 1000 Genomes Project Consortium. ``A Global Reference for Human Genetic Variation''. Nature 2015. 562: 68-74. doi:10.1038/nature15393

Hall B, {\bf Quitadamo A}, Shi X. ``A Graph Community Approach for Constructing microRNA Networks''. Big Data Computing and Communications, 283-293. doi:10.1007/978-3-319-22047-5\_23

{\bf Quitadamo A}, Tian L, Hall B, Shi X. ``An Integrated Network of microRNA and Gene Expression in Ovarian Cancer''. BMC Bioinformatics 2015, 16(Suppl 5):S5. doi:10.1186/1471-2105-16-S5-S5

Tian L, {\bf Quitadamo A}, Lin F, Shi X. ``Methods for Population Based eQTL Analysis in Human Genetics''. Tsinghua Science and Technology, 2014. 19(6): 624-634. doi:10.1109/TST.2014.6961031

\section{\sc Social Media}

LinkedIn: \href{http://linkedin.com/in/andrewquitadamo}{linkedin.com/in/andrewquitadamo}

Github: \href{https://github.com/andrewquitadamo}{github.com/andrewquitadamo}

\end{resume}
\end{document}



